src/css/profile/mobile/changeable/common/header.less

```diff
@@ -87,6 +87,15 @@
 		font-stretch: condensed;
 		border-radius: 0;
 
+		&.ui-btn-circle {
+			border-radius: 52 * @unit_base;
```

Comment:
Please consider setting it to 50%. It could be the source of the problems in case of dimensions change or TV profile.